ReactWindows/ReactNative/Bridge/CatalystInstance.cs

```diff
@@ -12,11 +12,16 @@
 
 namespace ReactNative.Bridge
 {
+    /// <summary>
+    /// A higher level API on top of the <see cref="IJavaScriptExecutor" /> and module registries. This provides an
+    /// environment allowing the invocation of JavaScript methods.
+    /// </summary>
     class CatalystInstance : ICatalystInstance, IDisposable
```

Comment:
Remove IDisposable here.  We also need to rethink whether the ICatalystInstance interface should publicly expose IDisposable.  I think only the "creator" of the CatalystInstance should be aware that it is disposable, so we should remove that interface and clean up the broken public surfaces.